docs/changelog.md

```diff
@@ -108,6 +108,7 @@ permalink: /changelog/
 
 * **browser-icons**
   * Handles low-memory scenarios by reducing memory footprint.
+  * Ensures icons are not cached on the disk in private sessions.
```

Comment:
Can you move this changelog entry up to 10.0.0-SNAPSHOT (You'll need to rebase this branch)? :)